tr.org.pardus.pen.yaml

```
@@ -0,0 +1,26 @@
+app-id: tr.org.pardus.pen
+runtime: org.kde.Platform
+runtime-version: 6.7
+sdk: org.kde.Sdk
+command: pardus-pen
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - name: pardus-pen
+    buildsystem: meson
+    build-options:
+      append-path: /usr/libexec
+    config-opts:
+      - -Dqt=6
+      - -Dscreenshot=false
+    sources:
+      - type: archive
+        url: https://github.com/pardus/pardus-pen/archive/refs/tags/debian/3.0.0.tar.gz
+        sha256: 155c423eb59a4cf282b0199f2af4137f796f53716090166d74085a0063a33d45
+    post-install:
+      - glib-compile-schemas /app/share/glib-2.0/schemas
+      - install data/tr.org.pardus.pen.appdata.xml  /app/share/metainfo/tr.org.pardus.pen.metainfo.xml
```

```suggestion
      - install ../data/tr.org.pardus.pen.appdata.xml  /app/share/metainfo/tr.org.pardus.pen.metainfo.xml
```

But then 
```
Installing /run/build/pardus-pen/data/tr.org.pardus.pen.appdata.xml to /app/metainfo
```
this is incorrect. Fixng it would make that unecessary.